src/ducks/Watchlists/Cards/Featured.js

```diff
@@ -1,25 +1,19 @@
 import React from 'react'
-import Card from './WatchlistCard'
-import { useFeaturedWatchlists } from '../gql/hooks'
-import { getWatchlistLink } from '../utils'
+import { WatchlistCards } from './Card'
+import { useFeaturedWatchlists } from '../gql/queries'
 
-const Featured = () => {
-  const [watchlists = [], loading] = useFeaturedWatchlists()
-  return !loading
-    ? watchlists.map(watchlist => {
-      return (
-        <Card
-          key={watchlist.id}
-          name={watchlist.name}
-          skipIndicator
-          watchlist={watchlist}
-          to={getWatchlistLink(watchlist)}
-          {...watchlist}
-          isWithNewCheck={false}
-        />
-      )
-    })
-    : null
+const FeaturedWatchlists = ({ className }) => {
+  const [watchlists] = useFeaturedWatchlists()
+
+  return (
+    <WatchlistCards
+      className={className}
+      watchlists={watchlists}
+      path='/watchlist/projects/'
+      isWithNewCheck={false}
```

Comment:
Maybe we can already return `isWithNewCheck` checking because we used it before for new featured lists